src/workerd/api/node/zlib-util.c++

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#include "zlib-util.h"
+
+#include "zlib.h"
+
+namespace workerd::api::node {
+
+namespace {
+// This function is required to avoid function name clash.
+uint32_t crc32Impl(uint32_t value, unsigned char* bytes, size_t size) {
+  return crc32(value, reinterpret_cast<const Bytef*>(bytes), size);
```

Comment:
nit: a comment here that Bytef is from zlib would be helpful I think. It's a bit obscure